unity-renderer/Assets/Scripts/MainScripts/DCL/Components/TextShape/TextShape.cs

```diff
@@ -62,8 +62,14 @@ public class Model : BaseModel
         public TextMeshPro text;
         public RectTransform rectTransform;
         private Model cachedModel;
+        private Material cachedFontMaterial;
```

Comment:
I think probably a better name would be `fontMaterial` removing de `cached` part

Reply:
This can conflate with `text.fontMaterial` making the code harder to read.